front/app/components/UI/FilterTabs/index.tsx

```diff
@@ -1,31 +1,11 @@
 import React, { useRef, KeyboardEvent } from 'react';
 import { ScreenReaderOnly } from 'utils/a11y';
 import styled from 'styled-components';
-import {
-  fontSizes,
-  isRtl,
-  colors,
-  media,
-} from '@citizenlab/cl2-component-library';
+import { fontSizes, colors, media } from '@citizenlab/cl2-component-library';
 import { rgba } from 'polished';
 import { FormattedMessage } from 'utils/cl-intl';
 import { MessageDescriptor } from 'react-intl';
-
-const TabsContainer = styled.div`
```

Comment:
Removed this since now the container comes from the HorizontalScroll component itself (so the necessary styling for hiding scroll bars etc. is applied within that component).